libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -40,7 +40,7 @@ bytes SortitionParamsChange::rlp() const {
   s << interval_efficiency;
   s << actual_correction_per_percent;
 
-  return s.out();
+  return s.invalidate();
```

Comment:
invalidate() return std::move(m_out). What's the benefits using invalidate? I think that's same here. I have no objection here, just want to learn.

Reply:
`out()` is just sending the reference to `s.m_out`  witch will probably creates copy on return. `invalidate()` should "steal" by moving `m_out`and return this data